qutip/models/dicke.py

```diff
@@ -0,0 +1,1325 @@
+"""
```

Comment:
QuTiP license stuff needs to be included at the top of the file. You can add yourselves to the list of authors for these files